### PR TITLE
Fixed typographical error, changed admitedly to admittedly in README.

### PR DESCRIPTION
**Pull request by orthographic-pedant**
@Morgul, I've corrected a typographical error in the documentation of the [omega](https://github.com/Morgul/omega) project. You should be able to merge this pull request automatically. However, if this was intentional or if you enjoy living in linguistic squalor, please let me know and [create an issue](https://github.com/thoppe/orthographic-pedant/issues/new) on my home repository.

### CODE REVIEW

**Comment by Morgul** [comment]
@orthographic-pedant **THANK U 4 DA ENLIGHTENMENT U HAS BROUGHT 2 MAH REPOSITORY. IM VRY HAPPEH 2 MERGE DIS CHANGE IN. U, SIR, R CEILIN CAT AMONG DOODZ.**

_(brought to you by http://speaklolcat.com/)_

**Comment by orthographic-pedant** [comment]
I'm sorry I wasn't very clear the first time. Let me try again...

@MORGUL, I HAS CORRECTD TYPOGRAFICAL ERROR IN DA DOCUMENTASHUN OV TEH OMEGA PROJECT. U SHUD BE ABLE 2 MERGE DIS PULL REQUEST AUTOMATICALLY. HOWEVR, IF DIS WUZ INTENSHUNAL OR IF U ENJOY LIVIN IN LINGUISTIC SQUALOR, PLZ LET ME KNOE AN CREATE AN ISSUE ON MAH HOME REPOSITORY.

Please don't make me do that again.
